docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -2415,7 +2415,7 @@ npx @material-ui/codemod@next v5.0.0/jss-to-styled <path>
 +  cta: `${PREFIX}-cta`,
 +  content: `${PREFIX}-content`,
 +}
-+const Root = styled('div')((theme) => ({
++const Root = styled('div')(({theme}) => ({
```

Comment:
@siriwatknp Was this diff created with the actual codemod? If so, then theme destructuring is probably missing in the codemod.

Reply:
my only concern with destructuring `theme` when using `styled` api is that it is inconsistent with using `css` prop in emotion where `theme` is passed directly without destructuring.

```tsx
import { Fragment } from 'react';
import { css } from '@emotion/react';
import styled from '@emotion/styled';
import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
import { darken } from '@material-ui/core/styles';
import Button from '@material-ui/core/Button';

const StyledButton = styled(Button)`
  ${({ theme }) => `
  color: ${theme.palette.common.white};
  background-color: ${theme.palette.info.main};
  &:hover {
    background-color: ${darken(theme.palette.info.main, 0.2)};
  }
  `}
`;

function Mui5Emotion(): EmotionJSX.Element {
  return (
    <Fragment>
      <Button
        css={(theme) => css`
          color: ${theme.palette.common.white};
          background-color: ${theme.palette.info.main};
          &:hover {
            background-color: ${darken(theme.palette.info.main, 0.2)};
          }
        `}
      >
        Button (css prop)
      </Button>
      <StyledButton>Button (styled)</StyledButton>
    </Fragment>
  );
}

export default Mui5Emotion;
```

If possible, I would prefer if we can destructure `theme` using both `css` prop and `styled` api:

```tsx
// prefer destructuring theme in both css prop & styled api for consistency
{({theme}) => ()}

// don't like doing this for styled api
{(theme) => ()}
// and doing this for css prop
{({theme}) => ()}
```

I'm not sure if this is possible, but if it is then I feel it should be consistent in both `css` prop & `styled` api. Also, I think destructuring is preferable than passing in `theme` directly to emotion

Reply:
> @siriwatknp Was this diff created with the actual codemod? If so, then theme destructuring is probably missing in the codemod.

codemod is correct (with destructuring) but the docs is not.

Reply:
@kimbaudi this PR is handling updates on the docs, not the actual API of the utilities. Please open an issue if there is a problem with some API.

Reply:
I would add empty spaces around the `theme` argument:
```suggestion
+const Root = styled('div')(({ theme }) => ({
```